packages/api/prisma/schema.prisma

```diff
@@ -15,13 +15,12 @@ datasource db {
 
 model Book {
   id     Int     @id
-  name   String
-  search String  @unique // This gives us a human readable key to look up books from URL params.
+  name   String  @unique
   verses Verse[]
 }
 
 model Verse {
-  id           Int                @id
+  id           String             @id
```

Comment:
I decided to make the verse ID follow a pattern which makes String a better fit. That pattern is: `BBCCCVVV` (B - book id, C - chapter number, V - verse number). Words use the same pattern but add two digits for the word order, which means we don't need an order column to sort words since the ID already has that information.

Reply:
I think we should start a docs file for the database structure. I do document this in the open api specs in the other PR, but I think it would be helpful to document the database schema itself.